From weiyi.huang@gmail.com  Fri Dec 18 10:13:32 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sun, 13 Dec 2009 21:13:54 +0800
Subject: Staging: dream: remove duplicated #include
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Huang Weiyi <weiyi.huang@gmail.com>
Message-ID: <1260710034-5588-1-git-send-email-weiyi.huang@gmail.com>


Remove duplicated #include('s) in
  drivers/staging/dream/smd/smd_rpcrouter.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/dream/smd/smd_rpcrouter.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/dream/smd/smd_rpcrouter.c
+++ b/drivers/staging/dream/smd/smd_rpcrouter.c
@@ -38,8 +38,6 @@
 #include <linux/platform_device.h>
 #include <linux/uaccess.h>
 
-#include <asm/byteorder.h>
-
 #include <mach/msm_smd.h>
 #include "smd_rpcrouter.h"
 
